components/invitation_modal/invitation_modal.jsx

```diff
@@ -231,90 +234,104 @@ export default class InvitationModal extends React.PureComponent {
+                        onCancel={this.state.confirmModal ? this.cancelClose : this.cancelBack}
+                    />
+                    {this.state.step === STEPS_INITIAL &&
+                    <InvitationModalInitialStep
+                        teamName={this.props.currentTeam.display_name}
+                        goToMembers={this.goToMembers}
+                        goToGuests={this.goToGuests}
+                        emailInvitationsEnabled={this.props.emailInvitationsEnabled}
+                    />
+                    }
+                    {this.state.step === STEPS_INVITE_MEMBERS &&
+                    <InvitationModalMembersStep
+                        teamName={this.props.currentTeam.display_name}
+                        currentTeamId={this.props.currentTeam.id}
+                        inviteId={this.props.currentTeam.invite_id}
+                        searchProfiles={this.props.actions.searchProfiles}
+                        emailInvitationsEnabled={this.props.emailInvitationsEnabled}
+                        onSubmit={this.onMembersSubmit}
+                        onEdit={this.onEdit}
+                    />
+                    }
+                    {this.state.step === STEPS_INVITE_GUESTS &&
+                    <InvitationModalGuestsStep
+                        teamName={this.props.currentTeam.display_name}
+                        currentTeamId={this.props.currentTeam.id}
+                        myInvitableChannels={this.props.invitableChannels}
+                        searchProfiles={this.props.actions.searchProfiles}
+                        searchChannels={this.props.actions.searchChannels}
+                        defaultChannels={this.state.lastInviteChannels}
+                        defaultMessage={this.state.lastInviteMessage}
+                        emailInvitationsEnabled={this.props.emailInvitationsEnabled}
+                        onSubmit={this.onGuestsSubmit}
+                        onEdit={this.onEdit}
+                    />
+                    }
+                    {this.state.step === STEPS_INVITE_CONFIRM &&
+                    <InvitationModalConfirmStep
+                        teamName={this.props.currentTeam.display_name}
+                        currentTeamId={this.props.currentTeam.id}
+                        onDone={this.close}
+                        onInviteMore={this.goToFirstStep}
+                        invitesType={this.state.invitesType}
+                        invitesSent={this.state.invitesSent}
+                        invitesNotSent={this.state.invitesNotSent}
+                    />
+                    }
+                </div>
+            </FullScreenModal>
+        </RootPortal>);
+
+        const actionsUserLimitModal = {
+            closeModal: this.props.actions.closeModal,
+            openModal: this.props.actions.openModal,
+        };
+        const userLimitModal = (<>
```

Comment:
Do you need the <> here?

Reply:
Not necessary. Removing it to get cleaner code.